operator/pkg/ciliumendpointslice/reconciler.go

```diff
@@ -94,7 +94,8 @@ func (r *reconciler) reconcileCESCreate(cesName CESName) (err error) {
 
 	for _, cepName := range ceps {
 		ccep := r.getCoreEndpointFromStore(cepName)
-		r.logger.Debug(fmt.Sprintf("Adding CEP to new CES (exist %v)", ccep != nil), logfields.CESName, cesName.string(),
+		r.logger.Debug(fmt.Sprintf("Adding CEP to new CES (exist %v)", ccep != nil),
```

Comment:
Tangentially related, `fmt.Sprintf` should be dropped and replaced with appropriate slog KVs (same for the other occurrences).